client/app/components/blueprint-details-modal/blueprint-details-modal-service.factory.js

```diff
@@ -41,6 +41,7 @@
 
       function resolveServiceCatalogs(CollectionsApi) {
         var options = {
+          mock: true,
```

Comment:
:exclamation:  WHY HAS THIS CHANGED?!   This doesn't seem to have anything to do with the new Rules functionality.   I don't want to go back to using mock data.  If this is merged it will mess up the Blueprints Details Dialog and the new Tagging I've implemented this sprint!!!!!